.prettierrc

```diff
@@ -0,0 +1,17 @@
+{
+	"plugins": [
+		"prettier-plugin-brace-style",
+		"prettier-plugin-space-before-function-paren",
+		"prettier-plugin-merge"
+	],
+	"braceStyle": "stroustrup",
+	"arrowParens": "avoid",
+	"bracketSpacing": true,
+	"endOfLine": "auto",
+	"semi": true,
+	"singleQuote": true,
+	"tabWidth": 4,
+	"useTabs": true,
+	"trailingComma": "all",
+	"printWidth": 100
```

Comment:
I see from the PR that runs Prettier that it's adding quotes around properties whether they are needed or not. I would suggest using the `preserve` setting: https://prettier.io/docs/options#quote-props

Reply:
Done